dnf5/main.cpp

```diff
@@ -246,6 +246,12 @@ void RootCommand::set_argument_parser() {
                 throw libdnf5::cli::ArgumentParserError(
                     M_("{}: Badly formatted argument value \"{}\""), std::string{"setopt"}, std::string(value));
             }
+
+            if (std::string(value) == std::string("installonly_limit=1"))
```

Comment:
Is this the best way to check for the installonly_limit setting in the config file?  Ideally we have the config file data in a better form so we're not having to compare against strings.  What if the string is "installonly_limit = 1"?